nav2_experimental/nav2_rl/nav2_turtlebot3_rl/turtlebot3_environment.py

```diff
@@ -24,6 +24,7 @@
 from geometry_msgs.msg import Twist, Pose
 from sensor_msgs.msg import LaserScan
 from gazebo_msgs.srv import GetEntityState
+from nav_msgs.msg import Path
```

Comment:
This is related to navigation task.  We should place it in `navigation_task_env`

Reply:
This is unused, so I removed it.